dashboard/templates/dashboard/index.html

```diff
@@ -3,20 +3,20 @@
 {% block content %}
     <!-- === Begin sd-latest-posts === -->
     <div id="sd-latest-posts-wrapper">
-        <section class="container">
-            <h2 class="text-center">Latest Posts</h2>
+        <div class="container">
             <div class="row">
-            <div class="col-md-12">
-                {% for post in user_posts %}
-                    <div id="post-{{ post.id }}" class="col-md-4">
-                        <div>{{ post.post_story }}</div>
-                        <div>{{ post.author.id }}</div>
-                    </div>
-                {% endfor %}
+            <!-- === Begin left-bar === -->
+            <div class="col-md-3">
+
             </div>
+            <!-- === End left-bar === -->
+            <!-- === Begin sd-posts === -->
+            {% include 'dashboard/includes/user_profile_posts.html' %}
```

Comment:
I was thinking everything would be in-line as one column. I assume you were thinking that maybe the other content would be, for example, the GitHub activity? Is there other content you were thinking of? I can't think of any other content at the moment that we would need to include.